app/controllers/review_mapping_controller.rb

```diff
@@ -51,129 +47,196 @@ def select_metareviewer
     @mapping = ResponseMap.find(params[:id])
   end
 
+
+  #EDITED: If the user is invalid(not present in database), then return with error message.
```

Comment:
Please make sure to skip a space before all open parens that do nob begin parameter lists.